template/src/utilities/networking.js

```diff
@@ -0,0 +1,80 @@
+import instance from "./axiosInstance";
+
+export function setHeader (property, data) {
+  instance.defaults.headers.common[property] = data;
+};
+
+export function setBaseURL (baseURL) {
+  instance.defaults.baseURL = baseURL;
+};
+
+export function setToken (token, type = null) {
+    switch (type) {
+      case "Paymentez": {
+        return (instance.defaults.headers.common["Auth-Token"] = token);
+      }
+
+      case "Bearer": {
+        return (instance.defaults.headers.common["Authorization"] = `Bearer ${token}`);
+      }
+
+      case "Basic": {
+        const { user, pass } = token;
+        return (instance.defaults.headers.common["Authorization"] = `Basic ${Buffer.from(
+          `${user}:${pass}`
+        ).toString("base64")}`);
+      }
+
+      default: {
+        return (instance.defaults.headers.common["Authorization"] = token);
+      }
+    }
+  };
+
+export function getRequest(api, config) {
+  return instance.get(api, config).then(response => {
+    return {data: response.data, status: response.status}
+  }).catch(error => {
+    return {data: null, status: error.response.status}
+  })
+}
+
+export function postRequest(api, body, config) {
+  if (typeof body === 'object' && body.constructor !== FormData) body = JSON.stringify(body)
+
+  return instance.post(api, body, config).then(response => {
+    return {data: response.data, status: response.status}
+  }).catch(error => {
+    return {data: null, status: error.response.status}
+  })
+}
+
```

Comment:
Willing to have postFormData function

Reply:
Hi @hoanganhle1996, This function was added. Please let me know if there are any problems